java-runtime-builder-app/src/main/resources/runtimes.properties

```diff
@@ -0,0 +1,3 @@
+# Mapping of runtime names to the digest of the current version
+openjdk=sha256:82b4389b81e45760198659e949e1414c84a555976edf204d9c8a5c4c1c0be571
```

Comment:
No longer used now that the base runtimes are passed in as cli arguments. This should be deleted.